common-libs/user-management/src/UserManagementService.ts

```diff
@@ -177,17 +194,25 @@ export class UserManagementService {
 
   async completeSignUpForEmailOrPhone(token: string, confirmationCode: string) {
     const { login, shortPassword } = this.parseSignUpToken(token)
-    this._loginShouldBeEmailOrPhoneNumber(login)
```

Comment:
@anton-iskryzhytskyi why we remove this check?

Reply:
Haw it was:
signUp token had a format 'login::password', then we derive username from login, we need username to complete sign-up
Now:
signUp token has a format 'username::password', username is uuid, so can't be email or phone